repo/meta/index.json

```diff
@@ -31,6 +31,18 @@
       "versions":{
         "alpha":"0"
       }
+    },
+    {
+      "currentVersion":"1.3.0",
+      "description":"Spark is a fast and general cluster computing system for Big Data",
+      "name":"spark",
+      "tags":[
+        "mesosphere",
+        "framework"
```

Comment:
They're just search terms, so anything goes!  Thanks for following precedent by tagging this package as a framework.